src/gascheduler.erl

```diff
@@ -92,7 +92,7 @@ start_link(Nodes, Client, MaxWorkers, MaxRetries) ->
 stop(Name) ->
     gen_server:call(Name, stop).
 
--spec execute(atom(), mfa()) -> ok.
+-spec execute(atom(), {module(), atom(), [term()]}) -> ok.
```

Comment:
There are other usages of `mfa()` type in this module that might also need to be changed.

Reply:
@lastres feel free to have a look and push to this branch. This sorted out a dialyze issue for gatransform, the more we fix, the better :)